torchxrayvision/utils.py

```diff
@@ -73,3 +77,41 @@ def load_image(fname: str):
     img = img[None, :, :]
 
     return img
+
+def read_xray_dcm(path:PathLike, voi_lut:bool=False, fix_monochrome:bool=True)->ndarray:
+    """read a dicom-like file and convert to numpy array 
+
+    Args:
+        path (PathLike): path to the dicom file
+        voi_lut (bool, optional): transform image to be human viewable. Defaults to False.
+        fix_monochrome (bool, optional): Convert dicom interpretation MONOCHROME1 to MONOCHROME2. Defaults to True.
+
+    Returns:
+        ndarray: 2D single array image for a dicom image
+    """
+    try:
+        import pydicom
+    except ImportError:
+        raise Exception("Missing Package Pydicom. Try installing it by running `pip install pydicom`.")
+
+    # get the pixel array
+    ds = pydicom.dcmread(path, force=True)
+    data = ds.pixel_array
+
```

Comment:
So I'm thinking something like this to safeguard users from images that we are not sure of yet. There are modes such as RGB, YBR_FULL, and YBR_FULL_422 and I don't think this code supports them right?

```suggestion
# we have not tested RGB, YBR_FULL, or YBR_FULL_422 yet.
if not ds.PhotometricInterpretation in ['MONOCHROME1', 'MONOCHROME2']:
   raise Exception(f'PhotometricInterpretation ({ds.PhotometricInterpretation}) not tested yet')

```

Reply:
Let me take a look how it impacts the currently implemented tests. Also, I have `ds.PhotometricInterpretation == ''RGB"` but I didnt use them because they are DICOM RGB Secondary Capture type. The type used for reports.

Also, I feel an exception might be a little harsh. How about an warning? The only specific task that is implemented here is the normalization step which is not specific to grayscale.

Reply:
I don't see how RGB images would be intentional for X-rays. 

> Also, I feel an exception might be a little harsh. How about an warning? 

I would prefer to save people from their disorganized data and make it really easy to catch errors. If someone is loading RGB X-ray images then they can load it using pydicom directly or open a ticket. It comes down to why we are implementing this function. I see it as a way for someone who is not an expert with dicom to read a dicom file and use the library. So if they do something that we can't verify works we should prevent a bug from happening in their code.

Reply:
> I see it as a way for someone who is not an expert with dicom to read a dicom file and use the library

Yup makes total sense for the scenario to raise an `Exception`.